packages/polyserve/src/args.ts

```diff
@@ -36,6 +36,14 @@ export const args: ArgDescriptor[] = [
     type: String,
     defaultValue: 'auto',
   },
+  {
+    name: 'do-not-compile',
```

Comment:
Now we have `--compile` and `--do-not-compile`, which might be slightly confusing. I guess we should just document the interaction between the two, especially that this takes precedence over `--compile=always`?

(I was also thinking we could update the `--compile` flag to take glob patterns (and minimatch already supports the `!` operator), but then you still have the `auto` vs `always` difference that needs to be represented somehow. Not sure if this idea makes sense.)

Reply:
Possible alternate names:

- `--pass-thru`
- `--do-not-modify`
- `--serve-as-is`
- `--as-is`
- `--no-transform`
- `--no-rewrite`
- `--immutable`
- `--static`
- `--constant`
- `--invariant`

Reply:
But do we *want* to disable all the transforms, or is this just about ES5 compilation?